DOCS/man/input.rst

```diff
@@ -2383,6 +2383,14 @@ Property list
     For accessing other information, like chapter start, see the
     ``chapter-list`` property.
 
+``track-metadata``
```

Comment:
What kind of track metadata is there? Is it something we might want to show in stats.lua page 5? (it display global metadata based on `display-tags`)

Reply:
What motivated this was `alpha_mode` being in the video track sample in #15553. I'm not sure what all would be useful that would show up in there, but at least that one might be.

Reply:
wouldn't it be more convenient to add tags to `track-list` node? It has all sort of things, so metadata can be there. Would be easier to find the correct tracks in one list.

Reply:
I'll be honest. I tried adding `track-list/0/medatada` at first but I gave up because `m_sub_property` doesn't support nodes and adding one seemed hard.

Reply:
I figured it out.